lbry/tests/unit/test_conf.py

```diff
@@ -207,6 +207,11 @@ def test_max_key_fee_from_yaml(self):
                 c.max_key_fee = None
             with open(config, 'r') as fd:
                 self.assertEqual(fd.read(), 'max_key_fee: null\n')
+            c = Config.create_from_arguments(
+                types.SimpleNamespace(config=config)
+            )
+            with open(config, 'r') as fd:
+                self.assertEqual(c.max_key_fee, None)
```

Comment:
Can you explain what's going on here? I don't understand what this is testing. (why the new `c` config instance and why the assert for None, also why that assert happens in a file context manager but the `fd` descriptor isn't used)

Reply:
This is trying to test `max_key_fee` is set to `None` that when config is created from `settings.yaml` where `max_key_fee: null\n` (the context manager shouldn't be there). It doesn't test my changes (or our code, for that matter), so I will remove it.